Sources/Hummingbird/Server/RequestContext.swift

```diff
@@ -136,6 +112,16 @@ extension HBRequestContext {
     public var id: String { self.logger[metadataKey: "hb_id"]!.description }
```

Comment:
I know this isn't in this PR, but I don't really like the way this `id` property is set up with a force unwrap.

Reply:
we can discuss this after, but I'm not keen on adding another field to the context, which isn't actually used in HB. I guess now if you want an id you can implement it yourself in your own context implementation